app/javascript/mastodon/features/getting_started/index.js

```diff
@@ -66,22 +66,23 @@ export default class GettingStarted extends ImmutablePureComponent {
       if (!columns.find(item => item.get('id') === 'PUBLIC')) {
         navItems.push(<ColumnLink key='3' icon='globe' text={intl.formatMessage(messages.public_timeline)} to='/timelines/public' />);
       }
+
+      navItems.push(<ColumnLink key='4' icon='question' text={intl.formatMessage(messages.keyboard_shortcuts)} to='/keyboard-shortcuts' />);
```

Comment:
I think this item should be displayed at the end of the navigation list. (After "Blocked users")